templates/test/ci/prow-clusterclass/cni-resource-set.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Are we supposed to have this line at the top?

Reply:
I think it's [needed](https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/9c74c714c2a7837f545ef484c0a3460fa0aa3c65/templates/test/ci/prow/cni-resource-set.yaml#L1) since we are adding a new yaml to the final result.